gateway-ha/src/main/java/io/trino/gateway/ha/config/HaGatewayConfiguration.java

```diff
@@ -225,4 +231,36 @@ public void setManagedApps(List<String> managedApps)
     {
         this.managedApps = managedApps;
     }
+
+    public List<String> getStatementPaths()
+    {
+        return statementPaths;
+    }
+
+    public void setAdditionalStatementPaths(List<String> statementPaths)
+    {
+        // remove trailing slashes to ensure predictable behavior when splitting on "/"
+        this.statementPaths = Streams.concat(ImmutableList.of(V1_STATEMENT_PATH).stream(),
+                statementPaths.stream().peek(s -> validateStatementPath(s, statementPaths)).map(s -> s.replaceAll("/$", ""))).toList();
```

Comment:
This only remove the last trailing slash, not all trailing slashes as stated in the comment.